### ISSUE TITLE
The Floating Label should conditionally have pointer-events:none

### ISSUE THREAD

**Issue by Xizario** [telerik/kendo-themes#3421]
Currently there is [rule](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/floating-label/_layout.scss#L33):
```css
.k-floating-label-container > .k-label{
pointer-events: none;
}
```
That prevents the label to be clicked. And from accessibility point of view the label should be clickable and the focus should move to the input that is associated with it.
However when there is text in the input with floating label, and the label stays above the input, clicking on the label does nothing. Can be reproduced here: just type some text into the input https://www.telerik.com/kendo-react-ui/components/labels/floating-label/ 

If we just remove the rule, then input can't be focused on mouseDown, so removing it is not an option.

My suggestion to move the `pointer-events: none;` into the [.k-floating-label-container.k-state-empty > .k-label](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/floating-label/_layout.scss#L43).
